src/components/buttons/ScrollUpButton/ScrollUpButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect, RefObject } from 'react';
```

Comment:
```suggestion
import { useState, useEffect, RefObject } from 'react';
```
`React` 이거는 빼도 될 것 같습니다~!